### PR TITLE
Upgrade kafka client to 1.1.0

### LINKED ISSUE #370: Upgrade Kafka Client to 1.1

### PR DESCRIPTION
**Pull request by sobychacko**
Upgrade kafka client to 1.1.0 for both kafka and kafka-streams binders

Resolves #370

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +47,12 @@
 				<artifactId>kafka-clients</artifactId>
 				<version>${kafka.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>org.apache.kafka</groupId>
+				<artifactId>kafka-clients</artifactId>
+				<version>${kafka.version}</version>
+				<classifier>test</classifier>
+			</dependency>
```
Comment (garyrussell) [inline pom.xml]: I think this just needs to be in test scope; looks like a bug in our docs.

File: pom.xml [hunk 2]
```diff
@@ -101,6 +107,27 @@
...
+					<exclusion>
+						<groupId>org.slf4j</groupId>
+						<artifactId>slf4j-log4j12</artifactId>
+					</exclusion>
+					<exclusion>
+						<groupId>log4j</groupId>
+						<artifactId>log4j</artifactId>
+					</exclusion>
+				</exclusions>
+			</dependency>
```
Comment (garyrussell) [inline pom.xml]: Why is this one in the parent but kafka itself is in the child poms?

File: pom.xml (hunk 1 above)
Comment (sobychacko) [reply pom.xml]: will address this.

File: pom.xml (hunk 2 above)
Comment (sobychacko) [reply pom.xml]: The ones in the parent pom are under the `dependencyManagment` section (Both kafka and the one with the `test` classifier). They both are in the relevant `dependencies` configuration in the child poms. Is that what you meant?

Comment (garyrussell) [reply pom.xml]: Yeah - I don't really know maven well enough to comment; it just looks a bit odd. I'll take your word if it's correct.

**Pull request merged**